### PR TITLE
Config editor updates preview

### PR DESCRIPTION
**Pull request by annabaker**
# Description
New page for in-app config editor, covers how to get to it and its available features.

(Lissa has seen/approved of these changes)

# Reasons
Low discovery rates in available config editor features. This seeks to mitigate this.

### CODE REVIEW

File: jekyll/_cci2/config-editor.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+[CircleCI CLI]({{ site.baseurl }}/2.0/local-cli/).
+
+![Config Editor]({{ site.baseurl }}/assets/img/docs/config-editor-main.png)
+
+The benefits of using the CircleCI config editor include:
+
+- Automatic validation and error checking
+- Auto-completion suggestions and configuration tooltips specific to CircleCI configuration syntax
+- Education of CircleCI concepts
+- Easily accessible CircleCI documentation, designed to best-suit your needs
```
Comment (thomasr8) [inline jekyll/_cci2/config-editor.md:20]:
```suggestion
- Easily accessible CircleCI documentation, designed to best suit your needs
```

Comment (thomasr8) [reply jekyll/_cci2/config-editor.md:20]: Not sure if you need to hyphenate best suit, but I could be wrong.

**Review by thomasr8 — APPROVED** [review]
@annabaker : This is great! I had one minor comment, which I am not sure is correct or not, but feel free to ignore it if I am wrong.

File: jekyll/_cci2/config-editor.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+layout: classic-docs
+title: "Using the CircleCI In-app Configuration Editor"
+description: "Docs page on In-app configuration editor use and features"
+version:
+- Cloud
+- Server v2.x
+---
+
+A UI environment, the CircleCI config editor, is provided for users who wish to modify their configurations without the use of the
```
Comment (teesloane) [inline jekyll/_cci2/config-editor.md:10]: I don't think user's can use the local-cli to make changes to their configuration so much as just editing it in the text-editor?

File: jekyll/_cci2/config-editor.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+---
+layout: classic-docs
+title: "Using the CircleCI In-app Configuration Editor"
+description: "Docs page on In-app configuration editor use and features"
+version:
+- Cloud
+- Server v2.x
```
Comment (teesloane) [inline jekyll/_cci2/config-editor.md:7]: Is this feature available on server 2.x? If not, we'll want to remove this tag.

File: jekyll/_cci2/config-editor.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+![Tooltips]({{ site.baseurl }}/assets/img/docs/config-editor-tooltips.png)
+
+## Automatic validation
+
+The config editor will automatically validate your configuration yaml after every change.
+
+At the bottom of the editor, you can verify that your configuration is valid.
+
+For a valid configuration, you will see the following:
```
Comment (teesloane) [inline jekyll/_cci2/config-editor.md:58]: Can we combine this into a single paragraph?
image

**Review by teesloane — CHANGES_REQUESTED** [review]
Added a few comments for possible changes, but is otherwise good to go.

File: jekyll/_cci2/config-editor.md (hunk 4 above)
Comment (annabaker) [reply jekyll/_cci2/config-editor.md]: @teesloane good catch, thanks -- i modified it.

File: jekyll/_cci2/config-editor.md (hunk 3 above)
Comment (annabaker) [reply jekyll/_cci2/config-editor.md:7]: @teesloane double checking

File: jekyll/_cci2/config-editor.md (hunk 2 above)
Comment (annabaker) [reply jekyll/_cci2/config-editor.md:10]: @teesloane you could use the clif or validation, but yeah you're right it'd be more like comparing the use of the cli/an editor vs the config editor where you get validation and editing in one screen. I added next to this -- "and/or a text editor. Using the CircleCI configuration editor eases the developer experience by giving users the ability to modify their CI/CD processes in a more unified fashion."

wdyt?

Comment (teesloane) [reply jekyll/_cci2/config-editor.md:10]: Sounds good to me!

**Comment by Onlythegeeway87** [comment]
In jekyll/_cci2/config-editor.md
<https://github.com/circleci/circleci-docs/pull/5145#discussion_r592921474>:

> +version:

+- Cloud

+- Server v2.x

+---

+

+A UI environment, the CircleCI config editor, is provided for users
who wish to modify their configurations without the use of the

+[CircleCI CLI]({{ site.baseurl }}/2.0/local-cli/).

+

+Config Editor

+

+The benefits of using the CircleCI config editor include:

+

+- Automatic validation and error checking

+- Auto-completion suggestions and configuration tooltips specific to
CircleCI configuration syntax

+- Education of CircleCI concepts

+- Easily accessible CircleCI documentation, designed to best-suit your needs


⬇️ Suggested change

-- Easily accessible CircleCI documentation, designed to best-suit your needs

+- Easily accessible CircleCI documentation, designed to best suit your needs


------------------------------

In jekyll/_cci2/config-editor.md
<https://github.com/circleci/circleci-docs/pull/5145#discussion_r592921637>:

> +version:

+- Cloud

+- Server v2.x

+---

+

+A UI environment, the CircleCI config editor, is provided for users
who wish to modify their configurations without the use of the

+[CircleCI CLI]({{ site.baseurl }}/2.0/local-cli/).

+

+Config Editor

+

+The benefits of using the CircleCI config editor include:

+

+- Automatic validation and error checking

+- Auto-completion suggestions and configuration tooltips specific to
CircleCI configuration syntax

+- Education of CircleCI concepts

+- Easily accessible CircleCI documentation, designed to best-suit your needs


Not sure if you need to hyphenate best suit, but I could be wrong.

**Comment by Onlythegeeway87** [comment]
In jekyll/_cci2/config-editor.md:

> +version: +- Cloud +- Server v2.x +--- + +A UI environment, the CircleCI
config editor, is provided for users who wish to modify their
configurations without the use of the +[CircleCI CLI]({{ site.baseurl
}}/2.0/local-cli/). + +Config Editor + +The benefits of using the
CircleCI config editor include: + +- Automatic validation and error
checking +- Auto-completion suggestions and configuration tooltips specific
to CircleCI configuration syntax +- Education of CircleCI concepts +-
Easily accessible CircleCI documentation, designed to best-suit your needs
⬇️ Suggested change-- Easily accessible CircleCI documentation, designed to
best-suit your needs +- Easily accessible CircleCI documentation, designed
to best suit your needs

In jekyll/_cci2/config-editor.md:

> +version: +- Cloud +- Server v2.x +--- + +A UI environment, the CircleCI
config editor, is provided for users who wish to modify their
configurations without the use of the +[CircleCI CLI]({{ site.baseurl
}}/2.0/local-cli/). + +Config Editor + +The benefits of using the
CircleCI config editor include: + +- Automatic validation and error
checking +- Auto-completion suggestions and configuration tooltips specific
to CircleCI configuration syntax +- Education of CircleCI concepts +-
Easily accessible CircleCI documentation, designed to best-suit your needs

Not sure if you need to hyphenate best suit, but I could be wrong.

**Review by teesloane — APPROVED** [review]

**Pull request merged**